### ISSUE TITLE
Support newlines in variable repr/str

### ISSUE THREAD

**Issue by studgeek** [inducer/pudb#203]
A useful option enable printing newlines for variable display for str/repr.

Right now if a variable contains newlines they are shown as ?. So it looks like the user typed `repr(var)` in interactive mode. For example:
`a {?   b? }?`

It would be handy if the newlines could be printed. So it would look the same as just typing `var` in interactive mode (or `print repr(var)`). For example:

```
a {
  b
}
```

**Comment by asmeurer** [comment]
Agreed, although perhaps this should only happen when line wrapping is enabled (`w`), as you always want one line per variable when it is off, I'd think. And regardless, we can probably do better than `?` to represent a newline
